lib/rules/web/admin_console/4.4/yaml_page.xyaml

```diff
@@ -0,0 +1,62 @@
+click_yaml_tab:
+  params:
+    tab_name: YAML
+  action: click_tab
+open_view_sidebar:
+  params:
+    button_text: View sidebar
+  action: click_button
+  element:
+    selector:
+      xpath: //button[contains(.,'Schema')]
+close_breadcrumb_sidebar:
+  action: click_close_button
+  element:
+    selector:
+      xpath: //button[contains(.,'Schema')]
+    missing: true
```

Comment:
common suggestion is to split click operator and checking operation like we talked in previous PR

Reply:
For this case, I think it's ok to keep as it is